pkg/pgmodel/migrations/sql/idempotent/base.sql

```diff
@@ -1154,35 +1189,72 @@ GRANT EXECUTE ON FUNCTION SCHEMA_CATALOG.create_metric_view(text) TO prom_writer
 
 --------------------------------- Views --------------------------------
 
+
+CREATE OR REPLACE FUNCTION SCHEMA_CATALOG.metric_view()
+RETURNS TABLE(id int, metric_name text, table_name name, retention_period  interval,
+                         chunk_interval interval, label_keys text[], size text, compression_ratio numeric,
+                         total_chunks bigint, compressed_chunks bigint)
+AS $func$
+BEGIN
+        IF NOT SCHEMA_CATALOG.is_timescaledb_installed() THEN
+                    RETURN QUERY
+                    SELECT
+                        m.id,
+                        m.metric_name,
+                        m.table_name,
+                        SCHEMA_CATALOG.get_metric_retention_period(m.metric_name) as retention_period,
+                        NULL as chunk_interval,
```

Comment:
I think I'd cast my nulls here. I know it will do it implicitly on the way out, but to prevent misordering bugs and the like it seems like a good idea.